src/coreclr/dlls/mscordac/CMakeLists.txt

```diff
@@ -49,6 +49,9 @@ else(CLR_CMAKE_HOST_WIN32)
 
         if (CLR_CMAKE_HOST_ARCH_ARM OR CLR_CMAKE_HOST_ARCH_ARM64)
             set(JUMP_INSTRUCTION b)
+        elseif (CLR_CMAKE_HOST_ARCH_LOONGARCH64)
```

Comment:
I think whole changes in this file can be moved to https://github.com/dotnet/runtime/pull/62886 to make this PR sef-contained.

Reply:
Thanks.
Had moved.